### PR TITLE
Fix recents call data and use shared component

### PR DESCRIPTION
**Pull request by adamweeks**
Still todo: 

- [x] Add jest tests for new component
- [x] Use component in activity list system message
- [x] Fix message names

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview ready!

Built with commit f8f308588367fe2f5cea2e6b7ab5ace360cf1b78

https://deploy-preview-592--hopeful-hamilton-4090b9.netlify.com

**Comment by adamweeks** [comment]
@bzang ok, finally ready

File: packages/node_modules/@ciscospark/react-component-call-data-activity/src/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import messages from './messages';
+import {parseActivityCallData} from './helpers';
+
+const propTypes = {
+  actor: PropTypes.shape({
+    entryUUID: PropTypes.string
+  }).isRequired,
+  duration: PropTypes.number.isRequired,
+  isGroupCall: PropTypes.bool,
```
Comment (bzang) [inline packages/node_modules/@ciscospark/react-component-call-data-activity/src/index.js]: missing default props

Comment (adamweeks) [reply packages/node_modules/@ciscospark/react-component-call-data-activity/src/index.js]: Fixed. made it required 😆

**Pull request merged**